test/web-platform-tests/run-to-upstream-web-platform-test.js

```diff
@@ -0,0 +1,86 @@
+"use strict";
```

Comment:
No, that sounds like too much work. Browsers also have non-upstreamed tests and don't do that. I guess I can investigate later.